ng2-components/ng2-alfresco-login/src/components/alfresco-login.component.ts

```diff
@@ -49,14 +50,17 @@ export class AlfrescoLoginComponent {
     backgroundImageUrl: string;
 
     @Input()
-    providers: string ;
+    providers: string;
 
     @Output()
     onSuccess = new EventEmitter();
 
     @Output()
     onError = new EventEmitter();
 
+    @Output()
+    executeOutcome: EventEmitter<FormOutcomeEvent> = new EventEmitter<FormOutcomeEvent>();
```

Comment:
Needs to be renamed to something else, the 'execute outcome' is the Activiti concept